### PR TITLE
[1LP][RFR]Fix/update ssa_datastores

### PR DESCRIPTION
**Pull request by rlbabyuk**
Purpose or Intent
=================
{{pytest: -k test_run_datastore_analysis -vvv --long-running --use-provider complete}}

### CODE REVIEW

**Comment by rlbabyuk** [comment]
There are 2 tests failing due to 1529725 bz

File: cfme/tests/infrastructure/test_datastore_analysis.py [hunk 1]
```diff
@@ -30,130 +27,96 @@
...
-    for i, argvalue_tuple in enumerate(argvalues):
+    for index, argvalue_tuple in enumerate(argvalues):
         args = dict(zip(argnames, argvalue_tuple))
-        datastores = args['provider'].data.get('datastores', {})
-        if not datastores:
-            continue
+        datastores = args['provider'].data.datastores
         for ds in datastores:
-            if not ds.get('test_fleece', False):
+            if not ds.test_fleece:
```
Comment (mshriver) [inline cfme/tests/infrastructure/test_datastore_analysis.py]: **Question/Required**

Why drop the `.get` call here, opening this to an AttributeError exception?

**Comment by mshriver** [comment]
@rlbabyuk Looks good - consider keeping the `.get` call, then rebase (and squash please) and I'll merge. Ping me when you've updated. Nice work!!

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/infrastructure/test_datastore_analysis.py (hunk 1 above)
Comment (rlbabyuk) [reply cfme/tests/infrastructure/test_datastore_analysis.py]: done, restored .get  call

**Pull request merged**